src/Microsoft.DotNet.Tools.Compiler/Program.cs

```diff
@@ -388,12 +406,32 @@ private static bool CompileProject(ProjectContext context, string configuration,
                              runtimeContext.CreateExporter(configuration));
             }
 
-            return PrintSummary(diagnostics, sw, success);
+            return PrintSummary(diagnostics, sw, preconditions, success);
+        }
+
+        private static string ResolveCompilerName(ProjectContext context, Preconditions preconditions)
+        {
+            var compilerName = context.ProjectFile.CompilerName;
+            compilerName = compilerName ?? "csc";
```

Comment:
Yes. Since I am moving the known compilers in the Constants class, I can probably put this in there as well, right?